drivers/ieee802154/ieee802154_nrf5.c

```diff
@@ -153,7 +153,9 @@ static void nrf5_rx_thread(void *arg1, void *arg2, void *arg3)
 			pkt_len = rx_frame->psdu[0] -  NRF5_FCS_LENGTH;
 		}
 
-		__ASSERT_NO_MSG(pkt_len <= CONFIG_NET_BUF_DATA_SIZE);
+		#if defined(CONFIG_NET_BUF_DATA_SIZE)
+			__ASSERT_NO_MSG(pkt_len <= CONFIG_NET_BUF_DATA_SIZE);
+		#endif
```

Comment:
Compliance check passed, but shouldn't this `#if` have no indent at all?
```suggestion
#if defined(CONFIG_NET_BUF_DATA_SIZE)
		__ASSERT_NO_MSG(pkt_len <= CONFIG_NET_BUF_DATA_SIZE);
#endif
```

Reply:
It does complian for `ifdef` but may be not for `if`